content/applications/general/email_communication/email_domain.rst

```diff
@@ -83,81 +84,91 @@ and has not been altered during transport.
 should not impact the deliverability if the DMARC check fails.
 
-You can check the DMARC record of a domain name with a tool like
-`MXToolbox DMARC <https://mxtoolbox.com/DMARC.aspx>`_.
+You can check the DMARC record of a domain name with a tool like `MXToolbox DMARC <https://mxtoolbo
+x.com/DMARC.aspx>`_.
 
 If one of your partners, customer or vendor, uses DMARC and has defined one of these policies, the
 Odoo server cannot relay emails from this partner to your users.
 
-You need to :ref:`handle user notifications in Odoo <discuss_app/notification_preferences>`, or replace the 
-email address of the partner with a default email address.
+You need to :ref:`handle user notifications in Odoo <discuss_app/notification_preferences>`, or
+replace the email address of the partner with a default email address.
 
 .. _email_communication/SPFDKIM_common_providers:
 
-SPF, DKIM & DMARC documentation of common providers 
+SPF, DKIM & DMARC documentation of common providers
 ===================================================
 
 - `OVH DNS <https://docs.ovh.com/us/en/domains/web_hosting_how_to_edit_my_dns_zone/>`_
 - `OVH SPF <https://docs.ovh.com/us/en/domains/web_hosting_the_spf_record/>`_
 - `GoDaddy TXT record <https://www.godaddy.com/help/add-a-txt-record-19232>`_
 - `GoDaddy SPF <https://www.godaddy.com/help/add-an-spf-record-19218>`_
 - `GoDaddy DKIM <https://www.godaddy.com/help/add-a-cname-record-19236>`_
-- `NameCheap <https://www.namecheap.com/support/knowledgebase/article.aspx/317/2237/how-do-i-add-txtspfdkimdmarc-records-for-my-domain/>`_
+- `NameCheap <https://www.namecheap.com/support/knowledgebase/article.aspx/317/2237/how-do-i-add-
+  txtspfdkimdmarc-records-for-my-domain/>`_
 - `CloudFlare DNS <https://support.cloudflare.com/hc/en-us/articles/360019093151>`_
 - `Google Domains <https://support.google.com/domains/answer/3290350?hl=en>`_
 - `Azure DNS <https://docs.microsoft.com/en-us/azure/dns/dns-getstarted-portal>`_
 
-To fully test your configuration, the tool `Mail-Tester <https://www.mail-tester.com/>`_ will give 
-you a full overview of the content and configuration you have in one email sent! Mail-Tester can 
+To fully test your configuration, the tool `Mail-Tester <https://www.mail-tester.com/>`_ will give
+you a full overview of the content and configuration you have in one email sent! Mail-Tester can
 also be used for other lesser known providers.
 
 Use a default email address
 ===========================
 
-To force the email address from which emails are sent, you need to create the following key in
-the System Parameters of the database:
+To force the email address from which emails are sent, a combination of the following keys needs to
+be set in the system parameters of the database:
+
+- `mail.default.from`: accepts the local part or a complete email address as value
+- `mail.default.from_filter`: accepts a domain name or a full email address as value
```

Comment:
`mail.default.from_filter` is only if you use the odoo-bin config, otherwise it's the from_filter field on ir.mail_server, not sure if it should me mentioned here